package.json

```diff
@@ -38,8 +38,13 @@
     "webpack-dev-server": "^3.10.3"
   },
   "dependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
     "@babel/polyfill": "^7.8.7",
+    "@babel/preset-typescript": "^7.9.0",
+    "prettier": "^2.0.4",
```

Comment:
is prettier still useful when we have eslint in place?

Reply:
they perform different task 

- Prettier is an opinionated code formatter
- ESLint is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code.